docs/notebooks/Calculating ground states on large scale systems with the quantum krylov subspace method.ipynb

```diff
@@ -0,0 +1,1037 @@
+    "    \\ket{0} \\ket{0}^N\\xrightarrow{H}&\\frac{1}{\\sqrt{2}}\n",
+    "    \\left(\n",
+    "    \\ket{0}\\ket{0}^N+ \\ket{1} \\ket{0}^N\n",
+    "    \\right)\\\\\n",
+    "    \\xrightarrow{\\text{1-ctrl-init}}&\\frac{1}{\\sqrt{2}}\\left(|0\\rangle|0\\rangle^N+|1\\rangle|\\psi_0\\rangle\\right)\\\\\n",
+    "    \\xrightarrow{U}&\\frac{1}{\\sqrt{2}}\\left(e^{i\\phi}\\ket{0}\\ket{0}^N+\\ket{1} U\\ket{\\psi_0}\\right)\\\\\n",
+    "    \\xrightarrow{\\text{0-ctrl-init}}&\\frac{1}{\\sqrt{2}}\n",
+    "    \\left(\n",
+    "    e^{i\\phi}\\ket{0} \\ket{\\psi_0}\n",
+    "    +\\ket{1} U\\ket{\\psi_0}\n",
+    "    \\right)\\\\\n",
+    "    =&\\frac{1}{2}\n",
+    "    \\left(\n",
+    "    \\ket{+}\\left(e^{i\\phi}\\ket{\\psi_0}+U\\ket{\\psi_0}\\right)\n",
+    "    +\\ket{-}\\left(e^{i\\phi}\\ket{\\psi_0}-U\\ket{\\psi_0}\\right)\n",
+    "    \\right)\\\\\n",
+    "    =&\\frac{1}{2}\n",
+    "    \\left(\n",
+    "    \\ket{+i}\\left(e^{i\\phi}\\ket{\\psi_0}-iU\\ket{\\psi_0}\\right)\n",
+    "    +\\ket{-i}\\left(e^{i\\phi}\\ket{\\psi_0}+iU\\ket{\\psi_0}\\right)\n",
+    "    \\right)\n",
+    "\\end{split}\n",
+    "\\end{equation}\n",
+    "\n",
+    "where we have used the classical simulable phase shift $ U\\ket{0}^N = e^{i\\phi}\\ket{0}$ in the third line. Therefore the expectation values are obtained as\n",
+    "\n",
+    "\\begin{equation}\n",
+    "\\begin{split}\n",
+    "    \\langle X\\otimes P\\rangle&=\\frac{1}{4}\n",
+    "    \\Big(\n",
+    "    \\left(e^{-i\\phi}\\bra{\\psi_0}+\\bra{\\psi_0}U^\\dagger\\right)P\\left(e^{i\\phi}\\ket{\\psi_0}+U\\ket{\\psi_0}\\right)\n",
+    "    \\\\\n",
+    "    &\\qquad-\\left(e^{-i\\phi}\\bra{\\psi_0}-\\bra{\\psi_0}U^\\dagger\\right)P\\left(e^{i\\phi}\\ket{\\psi_0}-U\\ket{\\psi_0}\\right)\n",
+    "    \\Big)\\\\\n",
+    "    &=\\text{Re}\\left[e^{-i\\phi}\\bra{\\psi_0}PU\\ket{\\psi_0}\\right],\n",
+    "\\end{split}\n",
+    "\\end{equation}\n",
+    "\n",
+    "\\begin{equation}\n",
+    "\\begin{split}\n",
+    "    \\langle Y\\otimes P\\rangle&=\\frac{1}{4}\n",
+    "    \\Big(\n",
+    "    \\left(e^{-i\\phi}\\bra{\\psi_0}+i\\bra{\\psi_0}U^\\dagger\\right)P\\left(e^{i\\phi}\\ket{\\psi_0}-iU\\ket{\\psi_0}\\right)\n",
+    "    \\\\\n",
+    "    &\\qquad-\\left(e^{-i\\phi}\\bra{\\psi_0}-i\\bra{\\psi_0}U^\\dagger\\right)P\\left(e^{i\\phi}\\ket{\\psi_0}+iU\\ket{\\psi_0}\\right)\n",
+    "    \\Big)\\\\\n",
+    "    &=\\text{Im}\\left[e^{-i\\phi}\\bra{\\psi_0}PU\\ket{\\psi_0}\\right].\n",
+    "\\end{split}\n",
+    "\\end{equation}\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Decompose time-evolution operator with Suzuki-Trotter decomposition\n",
+    "Instead of implementing the time-evolution operator exactly we can use the Suzuki-Trotter decomposition to implement an approximation of it. Repeating several times a certain order Trotter decomposition gives us further reduction of the error introduced from the approximation."
```

Comment:
A reference should be given for the Suzuki-Trotter decomposition. You have already introduced this decomposition earlier but did not mention it.

Reply:
added reference in the first place where the Suzuki-Trotter decomposition appears